blog/posts/basics-performance.md

```diff
@@ -0,0 +1,113 @@
+---
+title: Measuring page performance
+date: 2020-09-30
+author: Giovanni Rago
+githubUser: ragog
+tags: 
+  - basics
+  - performance
+---
+
+Over the course of recent years, the need for fast, responsive applications has been precipitated by the fact that web access has [largely shifted away](https://gs.statcounter.com/platform-market-share/desktop-mobile-tablet/worldwide/2019) from faster and better connected desktop machines to less powerful, slower mobile devices. 
```

Comment:
I would not use 'precipitated'. Just make it simpler, e.g. "The need for fast and responsive applications has never been greater because of the move from desktop to mobile" or something similar. It feels a bit to academic now.